src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1235,6 +1236,46 @@ typedef struct {
                      //!< RTCDataChannel object with the same id as the other peer.
 } RtcDataChannelInit, *PRtcDataChannelInit;
 
+
+/////////////////////////////////////////////////////
+/// Metrics/Stats Related structures
+/////////////////////////////////////////////////////
+
+/**
+ * @brief Collection of ICE related stats
+ */
+typedef struct {
+    PRTCIceServerStats pIceServerStats; //!< Server related stats. Reference in Stats.h
```

Comment:
Use RtcBlahBlah - capitalization here and everywhere else

Reply:
Done